config.sample.toml

```diff
@@ -131,7 +133,15 @@ exposure_interval = 30
 # to all worker nodes and removes inactive nodes from the central registry-server.
 cleanup_interval = 600
 deploy = true   # Deploy Kaze?
-port = 3000
+
+[services.kaze.mongodb]
+plugin = true  # Deploy MongoDB server and let `Kaze` manage it?
+container_port = 27019  # Port on which the MongoDB server container will run
+
+[services.kaze.redis]
+plugin = true  # Deploy Redis server and let `Kaze` manage it?
+container_port = 6380  # Port on which the Redis server container will run
```

Comment:
add environment variables for the user and password of both of them. See how it is done below for mongodb. Find the equivalent for redis